libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -90,6 +90,35 @@ public Optional<JourneyResponse> getJourneyResponseForStoredContraIndicators(
         return Optional.of(JOURNEY_RESPONSE_PYI_NO_MATCH);
     }
 
+    public boolean checkCiScoreBreaching(ContraIndicators contraIndicators)
+            throws UnrecognisedCiException {
+        LOGGER.info(
+                new StringMapMessage()
+                        .with(LOG_MESSAGE_DESCRIPTION.getFieldName(), "Retrieved user's CI items.")
+                        .with(
+                                LOG_NO_OF_CI_ITEMS.getFieldName(),
+                                contraIndicators.getContraIndicatorsMap().size()));
+        final int ciScore =
+                contraIndicators.getContraIndicatorScore(
+                        configService.getContraIndicatorConfigMap());
+        LOGGER.info(
+                new StringMapMessage()
+                        .with(LOG_MESSAGE_DESCRIPTION.getFieldName(), "Calculated user's CI score.")
+                        .with(LOG_CI_SCORE.getFieldName(), ciScore));
```

Comment:
Can we downgrade this log to `debug` (or omit entirely) - I don't think we should be logging this regularly.